nixos/modules/services/desktops/pipewire/wireplumber.nix

```diff
@@ -24,6 +47,103 @@ in
         description = lib.mdDoc "The WirePlumber derivation to use.";
       };
 
+      extraConfig = lib.mkOption {
+        # Two layer attrset is necessary before using JSON, because of the whole
+        # config file not being a JSON object, but a concatenation of JSON objects
+        # in sections.
+        type = lib.types.attrsOf (lib.types.attrsOf json.type);
+        default = { };
+        example = lib.literalExpression ''{
+          "log-level-debug" = {
```

Comment:
Do these examples still work on 0.5? Not saying they don't, but I also haven't checked ^^"

Reply:
I mentioned that above. The one with the headphone codec settings works, the log level one as well. The virtual item one I should maybe just drop, virtual items are cool but desktop Linux is not ready for them. The celluloid volume one I'm still fiddling with. I've tried a bunch of prop names, different rules sections and stared at pw-dump output, but no luck with that so far